dm/config/task.go

```diff
@@ -250,34 +250,48 @@ const (
 	LoadModePhysical = "physical"
 )
 
-// DuplicateResolveType defines the duplication resolution when meet duplicate rows.
+// DuplicateResolveType defines the duplication resolution when meet duplicate rows for logical import.
 type DuplicateResolveType string
 
 const (
 	// OnDuplicateReplace represents replace the old row with new data.
 	OnDuplicateReplace DuplicateResolveType = "replace"
 	// OnDuplicateError represents return an error when meet duplicate row.
-	OnDuplicateError = "error"
+	OnDuplicateError DuplicateResolveType = "error"
 	// OnDuplicateIgnore represents ignore the new data when meet duplicate row.
-	OnDuplicateIgnore = "ignore"
+	OnDuplicateIgnore DuplicateResolveType = "ignore"
+)
+
+// PhysicalDuplicateResolveType defines the duplication resolution when meet duplicate rows for physical import.
+type PhysicalDuplicateResolveType string
+
+const (
+	// OnDuplicateNone represents do nothing when meet duplicate row and the task will continue.
+	OnDuplicateNone PhysicalDuplicateResolveType = "none"
+	// OnDuplicateManual represents that task should be paused when meet duplicate row to let user handle it manually.
+	OnDuplicateManual PhysicalDuplicateResolveType = "manual"
```

Comment:
I remember we have three duplicate resolutions in lightning:
```
# Physical Import Mode 设置是否检测和解决重复的记录（唯一键冲突）。
# 目前支持三种解决方法：
#  - record: 仅将重复记录添加到目的 TiDB 中的 `lightning_task_info.conflict_error_v1` 表中。注意，该方法要求目的 TiKV 的版本为 v5.2.0 或更新版本。如果版本过低，则会启用下面的 'none' 模式。
#  - none: 不检测重复记录。该模式是三种模式中性能最佳的，但是可能会导致目的 TiDB 中出现数据不一致的情况。
#  - remove: 记录所有的重复记录，和 'record' 模式相似。但是会删除所有的重复记录，以确保目的 TiDB 中的数据状态保持一致。
# duplicate-resolution = 'none'
```
Are there any reasons we don't support it here?

Reply:
record is worse than remove in all respects so we delete it in design

Reply:
lgtm

Reply:
So we don't support "remove" in this version?

Reply:
"manual" is in fact "remove", it will remove the conflict KV first and let user query the recording table to decide to insert which row